tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "version": "1.8.2",
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es5",
+    "target": "es6",
```

Comment:
why?

Reply:
Because I was suddenly getting compiler errors, and Googling indicated that this was the only way to fix them.